src/socks-to-rtc/socks-to-rtc.ts

```diff
@@ -163,7 +163,10 @@ module SocksToRtc {
       this.onceStopped_ = this.onceStopping_.then(this.stopResources_);
       this.onceStopped_.then(this.dispatchEvent_.bind(this, 'stopped'));
 
-      return onceReady;
+      var rejectOnStopping = new Promise((F, R) => {
```

Comment:
Not related to this CL but I just realised the `onceStarted` field disappeared during the modularisation process. Not a big deal but sometimes it was nice to decouple it from the call to `start` -- particularly when we have >1 `start` methods.